src/Table/index.jsx

```diff
@@ -63,24 +63,28 @@ class Table extends React.Component {
   }
 
   getTableHeading(column) {
-    return (
-      this.props.tableSortable && column.columnSortable ?
-        <Button
-          label={
-            <span>
-              {column.label}
-              <span className={classNames(styles['sr-only'])}>
-                {' '}
-                {this.getSortButtonScreenReaderText(column.key)}
-              </span>
+    let heading;
+    if (this.props.tableSortable && column.columnSortable) {
+      heading = (<Button
```

Comment:
why not just `return` in each branch instead of assigning to a variable and then returning that variable?

Reply:
...errr and move the `if (column.hide)` branch first

Reply:
Regardless, is the intention that if there's a `column` with `columnSortable` = `true`, and `hide` = `true`, and `this.props.tableSortable` = `true` to render the column with a heading?

Reply:
> why not just return in each branch instead of assigning to a variable and then returning that variable?

I've always been a fan of the single entry, single exit philosophy of writing methods. It's not nearly as important of a principle here in js as it is in old-school C code, but I find it's still a benefit for debugging flow-of-execution without having any huge drawbacks.

> ...errr and move the if (column.hide) branch up

Any particular reason why? The way this table is currently being used, column headings are usually sortable and unhidden; hidden headings are somewhat unusual.

> Regardless, is the intention that if there's a column with columnSortable = true, and hide = true, and this.props.tableSortable = true to render the column with a heading? One could argue this is the fault of the API user for misusing the API, was just curious if this behavior was intentional.

Yeah, we talked about it and decided that it's non-sensical to hide a sortable column, as having a heading to click is a necessary condition of sorting a column. So `this.props.tableSortable && column.columnSortable` takes precedence, if that's true we can ignore the value of `hide` entirely.

Reply:
Cool, all makes sense.

My comment about moving the `column.hide` branch up was my thinking that the `hide` property would take precedence, but clearly that was intentionally not the case.